apps/website/src/components/EditPageLink.astro

```diff
@@ -18,14 +20,31 @@ if (page === 'components') {
 const repoUrl = 'https://github.com/iTwin/iTwinUI';
 const githubEditUrl = `${repoUrl}/edit/main/apps/website/src/pages/docs/${fileName}`;
 ---
+<div>
 
-<a href={githubEditUrl} {...rest}>
-  <EditIcon aria-hidden='true' />Edit this page on GitHub
-</a>
+  <a href={githubEditUrl} {...rest}>
+    <EditIcon aria-hidden='true' />Edit
+  </a>
+
+  <a href='https://github.com/iTwin/iTwinUI/discussions/new?category=documentation' {...rest}>
+    <ChatIcon aria-hidden='true' />Provide feedback
+  </a>
+
+  <a href='https://github.com/iTwin/iTwinUI/discussions/categories/documentation' {...rest}>
+    <ChatsIcon aria-hidden='true' />View pending feedback
+  </a>
```

Comment:
I don't think this link adds anything. Anyone who cares about pending discussions will already be subscribed to it. So lets remove it. Less cluttered that way.

After that, I would suggest using a column layout rather than row

Reply:
Removed **View pending feedback**.  I think for a first time browser of the site it would be nice for them to view all pending feedback to reduce duplicate feedback, but we can wait to hear that request from UX first.

Paragraph & title removed, links right aligned and vertical.